### ISSUE TITLE
Fighshare API spec possible bug

### ISSUE THREAD

**Issue by lukecoy** [mozillascience/software-discovery-dashboard#49]
It seems that the author resource representation returned by Figshares API may not be returning what is entirely expected. Figshare calls these representations _presenters_.

Example:
[Article #1394809](https://api.figshare.com/v1/articles/1394809) should correspond to [Figshare's Article Presenter Level 2](https://github.com/figshare/user_documentation/blob/master/APIv2/presenters/article.md#articlepresenterl2), which at a glance it seems to do (Article Presenter L2 = L1 & L2 combined). Article Presenter L2 also contains an Author Presenter inside it.

If we take a closer look at the [Author Presenter](https://github.com/figshare/user_documentation/blob/master/APIv2/presenters/author.md), it shows that ORCID is a field that you would expect to be returned, presumably if a specific author has an ORCID Figshare knows about. But Article #1394809 doesn't show an ORCID, even though the corresponding author, "Pedro Javier Villanueva Hernández" does have one on [his figshare public profile](https://figshare.com/authors/Pedro_Javier_Villanueva_Hern_ndez/726770). Seems like a bug.

[Another example with a different article](https://api.figshare.com/v1/articles/2059257)

TL;DR: There are multiple cases where Figshare's API isn't returning all the information I would expect. I think it probably means there's a bug in the API.

When this bug gets fixed (I'll make an issue on Fighshare's user_documentation repo), we should correct our author parsing for figshare to account for the change.

**Comment by lukecoy** [comment]
AH. API v1 doesn't specify ORCID. But V2 does. This isn't an issue, closing
